images/build/debian-iptables/Makefile

```diff
@@ -39,14 +38,13 @@ export DOCKER_CLI_EXPERIMENTAL := enabled
 SUDO=$(if $(filter 0,$(shell id -u)),,sudo)
 
 build:
-	cp -r ./$(CONFIG) $(TEMP_DIR)/
+	# Fix possible issues with the local umask
+	umask 0022
 
-ifneq ($(ARCH),amd64)
```

Comment:
What was are reasoning for not doing this on `amd64` in the past?

Reply:
We assumed that this was the native architecture, which is not necessarily always the case.

Reply:
Gotcha 👍